amgut/handlers/human_survey_completed.py

```diff
@@ -19,6 +80,15 @@ def get(self):
 
             self.render('human_survey_completed.html', skid=self.current_user,
                         surveys=surveys)
+            consent_info = ag_data.getConsent(human_survey_id)
+            message = build_consent_form(consent_info)
+
+            try:
+                send_email(message, 'American Gut - Signed Consent Form(s)',
+                           recipient=consent_info['participant_email'])
+            except:
+                Logger.exception('Error sending signed consent form for '
+                                 'survey ID: %s' % human_survey_id)
```

Comment:
possible to dump the associated email address, the exception type and the message from the exception?